packages/components/Tabs/src/TabsItem.settings.windows.ts

```diff
@@ -0,0 +1,79 @@
+import { tabsItemName, TabsItemType } from './TabsItem.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+
+export const settings: IComposeSettings<TabsItemType> = [
+  {
+    tokens: {
+      color: 'buttonText',
+      variant: 'heroStandard',
+      fontSize: 20,
+      borderWidth: 2,
+      borderColor: 'transparent',
+      borderRadius: 4,
+      indicatorColor: 'transparent',
+    },
+    root: {
+      accessible: true,
+      focusable: false,
+      // accessibilityRole: 'tab', // Add role when RN is at >= 0.64
```

Comment:
why?

Reply:
There is a [bug ](https://github.com/microsoft/react-native-windows/pull/6728) with react-native-windows 0.63 that makes elements focusable if they have an accessibilityRole, and thus not allowing the expected behavior on UWP.

Reply:
Thanks! That is very helpful to know =)